python/pyarrow/plasma.pyx

```diff
@@ -588,3 +606,66 @@ def connect(store_socket_name, manager_socket_name, int release_delay,
                               result.manager_socket_name,
                               release_delay, num_retries))
     return result
+
+def put(PlasmaClient client, list value, ObjectID object_id=None):
+    """
+    Store a Python value into the object store.
+
+    Parameters
+    ----------
+    client : PlasmaClient
+        The client connected to the object store we put the value in.
+    value : list
+        A Python object to store. Currently only lists are supported.
+    object_id : ObjectID, default None
+        If this is provided, the specified object ID will be used to refer
+        to the object.
+
+    Returns
+    -------
+    The object ID associated to the Python object.
+    """
+    cdef ObjectID id = object_id if object_id else ObjectID.from_random()
```

Comment:
I generally try to avoid using `id` as a variable name since it is also a built in python function